specification/sql/resource-manager/Microsoft.Sql/2017-03-01-preview/cancelOperations.json

```diff
@@ -0,0 +1,310 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-03-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/operations/{operationId}/cancel": {
+      "post": {
+        "tags": [
+          "Databases"
+        ],
+        "description": "Cancels the asynchronous operation on the database.",
+        "operationId": "Databases_Cancel",
```

Comment:
Sorry I didn't catch this sooner, but I feel this should be Databases_CancelOperation.  Did we decide against that?
@jiawang003 @jaredmoo